api/src/pcapi/routes/backoffice/offerers/forms.py

```diff
@@ -103,7 +103,7 @@ class Meta:
         csrf = False
 
     q = fields.PCOptSearchField(
-        "Nom de structure, SIREN, code postal, dép., ville, email, nom de compte pro, ID DMS CB"
+        "Nom d'entité juridique, SIREN, code postal, dép., ville, email, nom de compte pro, ID DMS CB"
```

Comment:
Ici on est dans la page « Entités juridiques à valider », raccourcissons car on ne voit pas tout sur un écran Full HD standard.
```suggestion
        "Nom, SIREN, code postal, dép., ville, email, nom de compte pro, ID DMS CB"
```
![image](https://github.com/user-attachments/assets/edd912d3-f586-4ecd-b1fd-ed638245f05d)